rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -1015,4 +1062,31 @@ public boolean changeStartAt(JobId jobId, String startAt)
         }
         return isJobStartAtChanged;
     }
+
+    @Override
+    public TaskFlowJob getInitialJobContent(JobId jobId)
```

Comment:
for now, the content is the one when you first submit the job, if the job is modified after, the modification will not be retrofitted in the job content, so i put initial as meaning the original job's content.